File: Linguagem C++/soma_vetor.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    int i, n,quant;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    double vet[n],soma, media;

    quant = 0;
    soma = 0;
    for(i = 0; i < n; i++)
    {
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
        soma = soma + vet[i];
        quant = quant + 1;
    }
    printf("Valores = ");
    for(i = 0; i < n; i++) {
        printf("%.1lf ", vet[i]);
    }

    printf("\n");

    media = soma / quant;
    printf("Soma = %.2lf \n", soma);
    printf("Media = %.2lf", media);

    return 0;
}

File: Linguagem C#/soma_vetor.cs
// See https://aka.ms/new-console-template for more information
using System.Globalization;
using System.Runtime.Serialization;
CultureInfo CI = CultureInfo.InvariantCulture;

Console.Write("Quantos numeros voce vai digitar? ");
int n = int.Parse(Console.ReadLine());

double[] vet = new double[n];
for (int i = 0; i < n; i++) {
    Console.Write("Digite um numero: ");
    vet[i] = double.Parse(Console.ReadLine(), CI);
}

Console.Write("valores = ");
for(int i = 0; i < n; i++) {
    Console.Write(vet[i].ToString("F1", CI)+ " ");
}

int quant = 0;
double soma = 0;
for(int i = 0; i < n; i++) {
    soma = soma + vet[i];
    quant++;
}
Console.WriteLine();
Console.WriteLine("Soma = " + soma.ToString("F2", CI));
Console.WriteLine("Media = " + (soma/quant).ToString("F2", CI));

